### PR TITLE
feat(frontend): support show config

### PR DESCRIPTION
**Pull request by yezizp2012**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

As title.

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:

- Summarize your change (**mandatory**)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Describe clearly one logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

If your pull request contains user-facing changes, please specify the types of the changes, and create a release note. Otherwise, please feel free to remove this section.

### Types of user-facing changes

Please keep the types that apply to your changes, and remove those that do not apply.

* SQL commands, functions, and operators
support show config like `SHOW QUERY_MODE`.

### Release note

Please create a release note for your changes. In the release note, focus on the impact on users, and mention the environment or conditions where the impact may occur.

### CODE REVIEW

File: src/common/src/session_config/mod.rs [hunk 1]
```diff
@@ -194,6 +194,24 @@ impl ConfigMap {
...
+        } else if key.eq_ignore_ascii_case(QueryMode::entry_name()) {
+            Ok(self.query_mode.to_string())
+        } else if key.eq_ignore_ascii_case(ExtraFloatDigit::entry_name()) {
+            Ok(self.extra_float_digit.to_string())
+        } else if key.eq_ignore_ascii_case(ApplicationName::entry_name()) {
+            Ok(self.application_name.to_string())
+        } else if key.eq_ignore_ascii_case(DateStyle::entry_name()) {
+            Ok(self.date_style.to_string())
+        } else {
+            Err(ErrorCode::UnrecognizedConfigurationParameter(key.to_string()).into())
```
Comment (BowenXiao1999) [inline src/common/src/session_config/mod.rs:211]: Prefer a match style

File: src/frontend/src/handler/variable.rs [hunk 2]
```diff
@@ -31,6 +33,29 @@ pub(super) fn handle_set(
...
 }
 
+pub(super) fn handle_show(context: OptimizerContext, variable: Vec<Ident>) -> Result<PgResponse> {
+    let config_reader = context.session_ctx.config();
+    if variable.len() != 1 {
+        return Err(
+            ErrorCode::InvalidInputSyntax("only one variable or ALL required".to_string()).into(),
+        );
+    }
+    // TODO: support SHOW ALL.
```
Comment (BowenXiao1999) [inline src/frontend/src/handler/variable.rs:43]: Track a issue number for this?

**Review by BowenXiao1999 — APPROVED** [review]
LGTM!

File: src/common/src/session_config/mod.rs (hunk 1 above)
Comment (yezizp2012) [reply src/common/src/session_config/mod.rs:211]: Well, it seems there's no way to apply match style here. 😭

**Pull request merged**